src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/ReturnsStepDef.scala

```diff
@@ -177,6 +177,69 @@ class ReturnsStepDef extends BaseStepDef {
         .navigateTo()
   }
 
+  When("I click on the change link to modify my contact details") { () =>
+    selectAccountingPeriod("Yes")
+    ReturnsPage
+      .provideUkRevenueInAccountingPeriod()
+      .selectAmlRegulatedActivity()
+      .submitPage()
+      .provideContactDetails("Oliver Tom", "Account Manager", "test@test.com", "01632 960 001")
+      .provideChangeContactDetails("James Bond 007", "Compliance Officer", "verify@oc.com", "014753777777")
```

Comment:
I don't understand this; you only change one aspect of the contact details at a time (depending on the change link that was clicked) so why provide all details here?